### PR TITLE
Added Wireframes

### PR DESCRIPTION
**Pull request by PunkChameleon**
Wireframe documentation for several user stories.

### CODE REVIEW

File: documentation/.$Wireframes.drawio.dtmp
Comment (RigatoniRex) [inline documentation/.$Wireframes.drawio.dtmp:1]: We can probably delete this temp file. Should also add this extension to the .gitignore file.

File: .gitignore [hunk 1]
```diff
@@ -161,3 +161,6 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Drawio temp files
+.$Wireframes.drawio.dtmp
```
Comment (PunkChameleon) [inline .gitignore:166]: @RigatoniRex -- good points, cleaned up!

**Review by RigatoniRex — APPROVED** [review]
Looks good!

**Review by vtattyba — APPROVED** [review]

**Review by RigatoniRex — CHANGES_REQUESTED** [review]
Just noticed that your wireframes use "first name" and "last name" instead of username, which is what we had in our ER diagram. We may want to resolve this first before we merge this.

We should either:
1. Modify the wireframe to use username
2. Update the ER to store first name/ last name

**Comment by PunkChameleon** [comment]
Good catch @RigatoniRex ! In my head, I'm not sure username makes sense if we can guid by email. We won't really need the username post login or do much with it. I'd vote the second of the two options, but I'm totally open if we think there's value for the username too.

**Comment by RigatoniRex** [comment]
> Good catch @RigatoniRex ! In my head, I'm not sure username makes sense if we can guid by email. We won't really need the username post login or do much with it. I'd vote the second of the two options, but I'm totally open if we think there's value for the username too.

Sounds good! I'll update the ER. We need to make sure we aren't using username anywhere else too.

**Review by RigatoniRex — APPROVED** [review]

**Pull request merged**